activecampaign.php

```diff
@@ -464,6 +464,12 @@ function activecampaign_getforms($ac, $instance) {
 function activecampaign_form_html($ac, $instance) {
 
 	if ($instance["forms"]) {
+		$domain = $instance["account"];
+		$protocol = "";
```

Comment:
I would say let's change this to be "https:" by default. Right now, we're using `//host/etc...` as our URL form and using `http://host/etc` if this is a CNAME. But we should probably use https if we can no matter what page you're visiting.